### PR TITLE
registering SAMstats 0.2.1 to bioconda repo

### PR DESCRIPTION
**Pull request by leepc12**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/samstats/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - SAMstatsParallel
+  commands:
+    - SAMstats --help
+    - SAMstatsParallel --help
+
+about:
+  home: The package home page
+  license: UNKNOWN
+  license_family: OTHER
+  license_file: ''
```
Comment (dpryan79) [inline recipes/samstats/meta.yaml]: This needs a license for us to legally redistribute it.

Comment (leepc12) [reply recipes/samstats/meta.yaml]: thanks, license information updated (MIT)

File: recipes/samstats/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+test:
+  imports:
+    - SAMstats
+    - SAMstatsParallel
+  commands:
+    - SAMstats --help
+    - SAMstatsParallel --help
+
+about:
+  home: The package home page
```
[the remark this answers is not in the document]
Comment (leepc12) [reply recipes/samstats/meta.yaml]: updated homepage, summary and details

**Comment by dpryan79** [comment]
Thanks @leepc12!

**Pull request merged**